### DISCUSSION TITLE
Turn on oversampling?

### DISCUSSION THREAD

**Discussion by davidm870** [artisan-roaster-scope/artisan#1433] [Q&A] [answered]
No doubt I've read past it several times, but how can I turn oversampling on?
Thanks!

**Comment by MAKOMO** [comment, marked as answer]
Oversampling has been removed from Artisan. It is not needed any longer.

**Reply by davidm870** [reply]
OK, then. Thanks!

**Reply by mwiercinski** [reply]
What is a recommended alternative to oversampling?

**Reply by roasterdave** [reply]
>Oversampling has been removed from Artisan. It is not needed any longer.

What problem are you trying to solve?

**Reply by MAKOMO** [reply]
Higher sampling rate combined with more filtering. Same effect.
